playbook/app/pb_kits/playbook/pb_dialog/_dialog.scss

```diff
@@ -227,6 +227,9 @@
 				height: 100%;
 				max-height: 100%;
 				max-width: none;
+				.dialog_body {
+					padding-bottom: $space_xl * 2 !important;
```

Comment:
Does this still work with our Global Spacing props. 
For example, if I use a complex dialog... and have a `<Dialog.Body paddingBottom="xs" />` which `!important` wins out?

Reply:
Good point, I'll pull down the branch again and test it out @jasperfurniss

Reply:
Great! Let me think through this one for a minute, and get back to you.

Reply:
Yes. Makes sense. I did notice the small scope, but still a little bit of a code smell, when our global props don't take specificity priority over something... Let me see if i can brainstorm a workaround.